yktrace/src/lib.rs

```diff
@@ -159,10 +159,11 @@ trait ThreadTracerImpl {
 /// behaviour.
 #[trace_head]
 pub fn start_tracing(kind: Option<TracingKind>) -> ThreadTracer {
-    match kind {
+    let th = match kind {
         Some(TracingKind::SoftwareTracing) => unimplemented!("software tracing is broken"),
         None | Some(TracingKind::HardwareTracing) => hwt::start_tracing()
-    }
+    };
+    th
```

Comment:
Is there a reason for this?

Reply:
Looks like a debugging left-over. Will remove.